DB/db.sql

```diff
@@ -121,10 +109,6 @@ INSERT INTO opera VALUES
 (16, './assets/nft16', 'Nome', 'DESC', 30.4, NULL),
 (17, './assets/nft17', 'Nome', 'DESC', 30.4, NULL);
```

Comment:
Nell'inserimento delle opera bisogna mettere admin al posto di NULL visto che ora il campo possessore è NOT NULL

Reply:
ops chiedo scusa, mi ero dimenticato